packages/lib-react-components/package.json

```diff
@@ -76,6 +76,7 @@
     "react-dom": "~16.8.6",
     "sinon": "~7.3.2",
     "sinon-chai": "~3.3.0",
+    "snazzy": "~8.0.0",
```

Comment:
I installed snazzy globally when I last worked on the linter, so it should already be installed in the root package file.

Reply:
I think this might be managed by yarn once we switch over, so I'm going to leave this here for now.